CONTRIBUTING.md

```diff
@@ -958,8 +958,9 @@ The pattern is simple:
    [cx-api/lib/features.ts](https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/cx-api/lib/features.ts).
    This map is inserted to generated `cdk.json` files for new projects created
    through `cdk init`.
-4. In your tests, use the `testFutureBehavior` and `testLegacyBehavior` [jest helper methods] to test the enabled and disabled behavior.
-5. In your PR title (which goes into CHANGELOG), add a `(under feature flag)` suffix. e.g:
+4. Add an entry for your feature flag in the [README](https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/cx-api/README.md) file.
+5. In your tests, use the `testFutureBehavior` and `testLegacyBehavior` [jest helper methods] to test the enabled and disabled behavior.
```

Comment:
So, now that we're on v2, these tests no longer work the way we expect them to. I realize your change isn't to this line, except to update the numbering, but could you edit this line to just say to test with and without the feature flags enabled?

Reply:
Yes, of course! Thanks for the hint. I struggled with this as well when I implemented a feature flag.

Reply:
I'm working on a rewrite for feature flags and the testing utilities but I don't yet have the design proposal done so it's going to be a minute. In the meantime I don't want to keep misleading people. Thanks for making this change for us!

Reply:
I was very pleased to do that. :)